docs/griptape-framework/drivers/file-manager-drivers.md

```diff
@@ -0,0 +1,34 @@
+---
+search:
+  boost: 2
+---
+
+## Overview
+
+File Manager Drivers can be used to load and save files with local or external file systems.
+
```

Comment:
The most common use-case of using these Drivers is probably with a File Loader. Can we add an example of this before we list out the Drivers?